nvidia_triton/tests/docker/info/v2

```diff
@@ -0,0 +1 @@
+{"name":"triton","version":"2.38.0","extensions":["classification","sequence","model_repository","model_repository(unload_dependents)","schedule_policy","model_configuration","system_shared_memory","cuda_shared_memory","binary_tensor_data","parameters","statistics","trace","logging"]}
```

Comment:
Duplicate file?

Reply:
Just one file here, it is required and it is named v2

Reply:
I guess is this file any different than the `nvidia_triton_metadata.json` file? Can you not mount that file instead in the caddyserver? Same with the metrics file.

Reply:
Oh I see.  Indeed I can use the same files